WikiDrips/ImageGenerator.swift

```diff
@@ -13,20 +13,12 @@ class ImageGenerator: Operation{
     let initials: String
     var image: UIImage?
     
-    init(initials: String){
+    init(initials: String) {
         self.initials = initials
     }
     
     override func main() {
-        if self.isCancelled {
-            print("Cancelled")
-            return
-        }
-        
-//        do {
-            self.image = UIImage.image(withInitials: initials)
-//        } catch {
-//            print(error)
-//        }
+        if isCancelled { print("Cancelled"); return }
+        image = UIImage.image(withInitials: initials)
     }
```

Comment:
There are several things Apple mentions that we need to do when subclassing Operation. Take look at:
https://developer.apple.com/reference/foundation/nsoperation
In particular, the sections on `KVO-Compliant Properties` and `Subclassing Notes`